static/js/constants.js

```diff
@@ -95,6 +95,18 @@ export const EMPLOYMENT_LEVEL = [
   "Prefer not to say"
 ]
 
+export const HIGHEST_EDUCATION_CHOICES = [
+  [1, "Doctorate"],
+  [2, "Master's or professional degree"],
+  [3, "Bachelor's degree"],
+  [4, "Associate degree"],
+  [5, "Secondary/high school"],
+  [6, "Junior secondary/junior high/middle school"],
+  [7, "Elementary/primary school"],
+  [8, "No formal education"],
+  [0, "Other education"]
+]
```

Comment:
@mbertrand Which of these values will get written out to hubspot? The main consumer of this data is going to be marketing, and I'd like the data to be easily understood by them.

Reply:
It would be the value stored in the database that gets sent to Hubspot.  In that case I'd recommend storing the user-friendly text instead of integers, assuming they are unlikely to change.

Reply:
Thanks @mbertrand. @pdpinch Could you provide the list of strings we want to store in database/hubspot.

Reply:
Thanks Matt. How about this then?

```
  ["Doctorate", "Doctorate"],
  ["Master's or professional degree", "Master's or professional degree"],
  ["Bachelor's degree", "Bachelor's degree"],
  ["Associate degree", "Associate degree"],
  ["Secondary/high school", "Secondary/high school"],
  ["Junior secondary/junior high/middle school", "Junior secondary/junior high/middle school"],
  ["Elementary/primary school", "Elementary/primary school"],
  [ "No formal education", "No formal education"],
  ["Other education", "Other education"]
```

And then I guess we have to change the serializer as well, to take a string instead of an integer.

Reply:
ok thanks, I will update the code according to the above list.

Reply:
Done.